pkg/helm/charts/tests/centralservices/flavor/flavor_test.go

```diff
@@ -52,30 +53,24 @@ func TestOverriddenTagsAreRenderedInTheChart(t *testing.T) {
 
 func TestWithDifferentImageFlavors(t *testing.T) {
 	testbuildinfo.SetForTest(t)
-	// having a function as value allows to successfully run this test without dependency to GOTAGS='' and GOTAGS='release'
-	imageFlavorCases := map[string]func() defaults.ImageFlavor{
-		"development": func() defaults.ImageFlavor {
-			testutils.SetVersion(t, testutils.GetExampleVersion(t))
-			return defaults.DevelopmentBuildImageFlavor()
-		},
-		"stackrox": func() defaults.ImageFlavor {
-			testutils.SetVersion(t, testutils.GetExampleVersionUnified(t))
-			return defaults.StackRoxIOReleaseImageFlavor()
-		},
-		"rhacs": func() defaults.ImageFlavor {
-			testutils.SetVersion(t, testutils.GetExampleVersionUnified(t))
-			return defaults.RHACSReleaseImageFlavor()
-		},
-		"custom": func() defaults.ImageFlavor {
-			return customFlavor(t)
-		},
+	testutils.SetVersion(t, testutils.GetExampleVersion(t))
+	imageFlavorCases := map[string]defaults.ImageFlavor{
+		"development": defaults.DevelopmentBuildImageFlavor(),
+		"stackrox":    defaults.StackRoxIOReleaseImageFlavor(),
+		"rhacs":       defaults.RHACSReleaseImageFlavor(),
+		"custom":      customFlavor(t),
+	}
+	if buildinfo.ReleaseBuild {
+		imageFlavorCases["opensource-release"] = defaults.OpenSourceImageFlavor()
+	} else {
+		imageFlavorCases["opensource-development"] = defaults.OpenSourceImageFlavor()
 	}
 
 	for name, f := range imageFlavorCases {
-		imageFlavor := f()
 		t.Run(name, func(t *testing.T) {
+			f := f
 			helmChartTestUtils.RunHelmTestSuite(t, testDir, image.CentralServicesChartPrefix, helmChartTestUtils.RunHelmTestSuiteOpts{
-				Flavor:       &imageFlavor,
+				Flavor:       &f,
```

Comment:
Could we keep the variable name as `imageFlavor`? Change it in the range variable as well just to be a bit more clear.

Reply:
Done.